### PR TITLE
Add possible values to indicate that a build is a production build

### PR DESCRIPTION
**Pull request by johnsonw**
- provide list of possible truthy values for a production build
- remove unnecessary environment variables

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: copr/create_build.py [hunk 1]
```diff
@@ -90,7 +87,7 @@ def write_new_spec(spec_file, new_data):
 try:
     p = glob.glob(srpm_path).pop()
 except:
-    if not prod:
+    if not prod in ["TRUE", "True", "true", "t", "T", "Y", "y", "YES", "Yes", "yes"]:
```
Comment (jgrund) [inline copr/create_build.py]: Is there some way to do this without enumerating all possible "yes" variants?

Comment (johnsonw) [reply copr/create_build.py]: I suppose we could only support `True` and convert it to a boolean, though it's probably less "safe". Is there a preferred method to use?

Comment (jgrund) [reply copr/create_build.py]: I can't think of another way to do this for env vars.

File: copr/create_build.py [hunk 2]
```diff
@@ -119,17 +76,37 @@ def write_new_spec(spec_file, new_data):
 
     p = glob.glob(srpm_path).pop()
 
-build = client.build_proxy.create_from_file(owner, project, p)
+if local_only in valid_truthy_args:
+    print("Building the RPM from SRPM Locally.")
+    subprocess.call(["rpmbuild", "--rebuild", p])
```
Comment (jgrund) [inline copr/create_build.py]: I think this should become:

```
subprocess.call(["rpmbuild", "--rebuild", p, "-D", "%_topdir /build/_topdir"], cwd="/build")
```

and then remove the glob and mv after this.

**Pull request merged**